VocaDbModel/DataContracts/Venues/VenueDetailsContract.cs

```diff
@@ -0,0 +1,20 @@
+using System.Linq;
+using VocaDb.Model.DataContracts.ReleaseEvents;
+using VocaDb.Model.Domain.Globalization;
+using VocaDb.Model.Domain.Venues;
+
+namespace VocaDb.Model.DataContracts.Venues {
+
+	public class VenueDetailsContract : VenueContract {

```

Comment:
This could be replaced with VenueForApiContract too, I think.

Reply:
Do you think `VenueForEditContract` should be replaced with `VenueForApiContract`, too?

Reply:
Perhaps I would keep VenueForEditContract separate. The edit use case is quite special.